pebble/pathapp/wscript

```diff
@@ -0,0 +1,40 @@
+
+#
+# This file is the default set of rules to compile a Pebble project.
+#
+# Feel free to customize this to your needs.
+#
+
+import os.path
+
+top = '.'
+out = 'build'
```

Comment:
They are used within the pebble [Waf builder](https://waf.io/apidocs/index.html) to denote the names for top-level and build directories respectively I guess.